src/app/app.module.ts

```diff
@@ -35,7 +35,7 @@ export function initAppFn(envService: AppConfigService) {
     TranscriptionsComponent,
     LoginComponent,
   ],
-  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, MatIconModule, HttpClientModule],
+  imports: [BrowserModule, AppRoutingModule, ReactiveFormsModule, MatIconModule, HttpClientModule, ReactiveFormsModule],
```

Comment:
`ReactiveFormsModule` us in there twice

Reply:
second instance of `ReactiveFormsModule` has been removed in latest commit